src/Codeception/Lib/Interfaces/Web.php

```diff
@@ -14,7 +14,7 @@
      * $I->amOnPage('/register');
      * ```
      *
-     * @param $page
+     * @param string $page
```

Comment:
I think Yii2 module should override this method as well as the docblock (and documentation) to change this behavior.